examples/05_assets/main.rs

```diff
@@ -13,8 +13,8 @@ use amethyst::config::Config;
 use amethyst::ecs::World;
 use amethyst::ecs::input::InputBundle;
 use amethyst::ecs::rendering::{AmbientColor, Factory, LightComponent, MaterialComponent,
-                               MeshComponent, MeshContext, RenderBundle, TextureComponent,
-                               TextureContext};
+MeshComponent, MeshContext, RenderBundle, TextureComponent,
```

Comment:
This looks like broken formatting, are you using the latest rustfmt-nightly? And running cargo +nightly fmt --all ?

Reply:
I don't really know what went on here either, I ran the latest nightly but I'll try again this afternoon.